packages/atomic/src/components/search/result-template-components/atomic-result-example-component/atomic-result-example-component.spec.ts

```diff
@@ -0,0 +1,11 @@
+import {describe, it, expect} from 'vitest';
+import './atomic-result-example-component.js';
+
+describe('atomic-result-example-component', () => {
```

Comment:
Should UTs also cover localization (at least broadly)?

Reply:
Probably yes, tbd